src/Controller/isInputEmpty.js

```diff
@@ -0,0 +1,8 @@
+const isInputEmpty = userInput => {
+  if (userInput === '') {
+    return true;
+  }
+  return false;
+};
+
+export default isInputEmpty;
```

Comment:
이 부분도 이렇게 줄일 수 있지 않을까요?

```suggestion

const isInputEmpty = userInput => userInput === '';
```